pytripgui/canvas_vc/plotter/managers/voi_manager.py

```diff
@@ -0,0 +1,200 @@
+import colorsys
+import logging
+
+import matplotlib.colors
+import numpy as np
+from matplotlib.axes import Axes
+from matplotlib.lines import Line2D
+from pytrip.vdx import Slice
+
+from pytripgui.canvas_vc.objects.vdx import Vdx
+from pytripgui.canvas_vc.plotter.managers import BlitManager
+
+logger = logging.getLogger(__name__)
+
+
+class VoiManager:
+    def __init__(self, axes: Axes, blit_manager: BlitManager):
+        self._axes: Axes = axes
+        self._blit_manager: BlitManager = blit_manager
+        self._plotted_voi: dict = {}
+
+        # colors

```

Comment:
Use colors consistent with `pytrip` definition, see following methods of `Vdx` class:
- `define_colors`
- `get_color`
- `set_color`

Code at:
https://github.com/pytrip/pytrip/blob/e2b4d22d5ebba50e5e3545d2e140a56982234ca7/pytrip/vdx.py#L824

The reasons of keeping colors at `pytrip` level are following:
 - these colors could be used in subsequent development of `cubeslicer` code (command line tools embedded in pytrip package which generates .png files for each slice found in CT and dose cube)
 - some logic can be added to color definition, i.e. typical planning volumes (i.e. CTV, PTV) could get some typical colors, i.e. red
--

Reply:
Done for now, but needs to be changed after changes in pytrip